config/initializers/ckeditor.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+Ckeditor.setup do |config|
+  # //cdn.ckeditor.com/<version.number>/<distribution>/ckeditor.js
+  config.cdn_url = "//cdn.ckeditor.com/4.19.1/standard/ckeditor.js"
```

Comment:
I'm not sure I follow this. Are we using version from here (4.19.1) or the one in the Gemfile (5.1.3)?

Reply:
The gem and ckeditor's versions are not really synced. Ckeditor was upgraded to version 5 in gem version 5.20. Here I've upgraded to the latest ckeditor (gem) version which supported v4.

But also I had to override the default version from the gem because ckeditor changed its license in 4.25 to a comercial one which is the latest of v4 and it's LTS. I'll leave a comment about that here